emitter.go

```diff
@@ -198,7 +191,7 @@ func CopyMulty(stop chan int, src io.Reader, writers ...io.Writer) error {
 					hasher := fnv.New32a()
 					// First 20 bytes contain tcp session
 					id := payloadID(payload)
-					hasher.Write(id[:20])
+					hasher.Write(id)
```

Comment:
Check return value.

Reply:
We won't really use it anywhere! it will unnecessary allocation